docs/manual-EN/1.overview/2.quick-start/4.import-csv-file.md

```diff
@@ -0,0 +1,243 @@
+
+```ngql
+CREATE TAG player (name string, age int);
+
+CREATE TAG team (name string);
+
+CREATE EDGE serve (start_year int, end_year int);
+
+CREATE EDGE follow (degree, int);
+```
+
+## Preparing Your Configuration File
+
+You must configure the `.yaml` configuration file, which regulates how data is organized in the CSV files. In this example we create a *config.yaml* file.
+
+In this example, we configure the `config.yaml` configuration file as follows:
+
+```ngql
+version: v1rc1
+description: example
+clientSettings:
+  concurrency: 2 # number of graph clients
+  channelBufferSize: 1
+  space: nba
+  connection:
+    user: user
+    password: password
+    address: 127.0.0.1:3699
+logPath: ./err/test.log
+files:
+  - path: /home/nebula/serve.csv
+    failDataPath: ./err/serve.csv
+    batchSize: 2
+    type: csv
+    csv:
+      withHeader: false
+      withLabel: false
+    schema:
+      type: edge
+      edge:
+        name: serve
+        withRanking: false
+        props:
+          - name: start_year
+            type: int
+          - name: end_year
+            type: int
+  - path: /home/nebula/follow.csv
+    failDataPath: ./err/follow.csv
+    batchSize: 2
+    type: csv
+    csv:
+      withHeader: false
+      withLabel: false
+    schema:
+      type: edge
+      edge:
+        name: follow
+        withRanking: false
+        props:
+          - name: degree
+            type: int
+  - path: /home/nebula/player.csv
+    failDataPath: ./err/player.csv
+    batchSize: 2
+    type: csv
+    csv:
+      withHeader: false
+      withLabel: false
+    schema:
+      type: vertex
+      vertex:
+        tags:
+          - name: player
+            props:
+              - name: name
+                type: string
+              - name: age
+                type: int
+  - path: /home/nebula/team.csv
+    failDataPath: ./err/team.csv
+    batchSize: 2
+    type: csv
+    csv:
+      withHeader: false
+      withLabel: false
+    schema:
+      type: vertex
+      vertex:
+        tags:
+          - name: team
+            props:
+              - name: name
+                type: string
+
+```
+
+**Note**:
+
+* In the above configuration file, you must change the IP address and the port number to yours.
+
+* You must change the directory of the CSV files to yours, otherwise **Nebula Importer** cannot find the CSV files.
+
+## Preparing the CSV Data
+
+In this example we prepare four CSV data files: **player.csv**, **team.csv**, **serve.csv**, and **follow.csv**.
+
+The data in the **serve.csv** file is as follows:
+
+```csv
+100,200,1997,2016
+101,201,1999,2018
+102,203,2006,2015
+102,204,2015,2019
+103,204,2017,2019
+104,200,2007,2009
+```
+
+The data in the **follow.csv** file is as follows:
+
+```csv
+100,101,95
+100,102,90
+101,100,95
+102,101,75
+102,100,75
+103,102,70
+104,101,50
+104,105,60
+105,104,83
+```
+
+The data in the **player.csv** file is as follows:
+
+```csv
```

Comment:
Let's make the demo data unified. I suggest using the full names here to make our example more vivid.

Reply:
As there is a space in a full name, I don't know how to deal with this tricky issue in csv files. So I make a compromise.

Reply:
I think it's ok with a space in the full name, you can try.